index.html

```diff
@@ -1,21 +1,195 @@
+        <div class="overlay">
+          <!-- Description Section  -->
+          <section class="description">
+            <h1>Hello</h1>
+            <h3>Here's who I am & what I do</h3>
+            <div class="btns">
+              <a class="btn btn1 btn-primary" href="resume.html">RESUME</a>
+              <a class="btn btn2 btn-secondary" href="Projects.html"
+                >PROJECTS</a
+              >
+            </div>
+            <p>
+              I'm a paragraph. Click here to add your own text and edit me. It’s
+              easy. Just click &ldquo; Edit Text &rdquo; or double click me to
+              add your own content and make changes to the font. <br /><br />
+              <span class="below-text"
+                >I’m a great place for you to tell a story and let your users
+                know a little more about you.</span
+              >
+            </p>
+          </section>
 
+          <!-- Introduction -->
+          <section class="introduction">
+            <div class="row">
+              <div class="col">
+                <img src="./img/maya.jpg" alt="Maya images" />
+              </div>
+              <div class="col">
+                <h2>
+                  <strong>
+                    Maya <br />
+                    Nelson</strong
+                  >
+                </h2>
+              </div>
+              <div class="col blue-line"></div>
+              <div class="col btns">
+                <a class="btn btn-primary" href="resume.html">RESUME</a>
+                <a class="btn btn-secondary" href="Projects.html">PROJECTS</a>
+              </div>
+              <p>PROJECT MANAGER</p>
+              <div class="col col-icons">
+                <ul class="social-icons">
+                  <li>
+                    <a href="https://www.facebook.com/wix"
+                      ><i class="fa-brands fa-facebook-f fa-xl"></i
+                    ></a>
+                  </li>
+                  <li>
+                    <a href="https://twitter.com/wix"
+                      ><i class="fa-brands fa-twitter fa-xl"></i
+                    ></a>
+                  </li>
+                  <li>
+                    <a href="https://www.linkedin.com/company/wix-com/"
+                      ><i class="fa-brands fa-linkedin-in fa-xl"></i
+                    ></a>
+                  </li>
+                  <li>
+                    <a href="https://www.instagram.com/wix/"
+                      ><i class="fa-brands fa-instagram fa-xl"></i
+                    ></a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </section>
+        </div>
+      </main>
 
-    <!-- Please leave this in to credit the creators of this design -->
-    <p>This website design was created by Wix.com, and is used here for strictly educational purposes.</p>
-</body>
+      <!-- Footer section -->
+      <footer>
+        <section class="rows all-divs">
```

Comment:
let's add padding-top to fix this overlap 

before
<img width="834" alt="스크린샷 2023-03-05 23 47 00" src="https://user-images.githubusercontent.com/63498427/222993011-c16e4721-2494-4511-994c-bfd0a6a45a98.png">


after
<img width="835" alt="스크린샷 2023-03-05 23 46 51" src="https://user-images.githubusercontent.com/63498427/222993020-2657437d-9dcf-48ad-a35d-911cc1f810a9.png">